contrail-controller/hooks/contrail_controller_hooks.py

```diff
@@ -155,6 +172,9 @@ def config_changed():
         if is_leader():
             _address_changed(local_unit(), ip)
 
+        if config.get('local-rabbitmq-hostname-resolution'):
```

Comment:
this will lead to updating /etc/hosts only with local IP/name. another logic is needed to be able to change this option on-the-fly.